pyproject.toml

```
@@ -76,7 +76,7 @@ shapely = { version = ">=1.7.0", optional = true }  # groups = ["user", "dev"]
 
 # Streamlit's pinning is due to its current beta nature and the fact that some
 # private APIs are utilised.
-streamlit = { version = "0.74.1", optional = true }  # groups = ["user", "dev"]
+streamlit = "^0.75.0"  # groups = ["user", "dev"]
```

Replace this with:

```suggestion
streamlit = { version = "0.75.0", optional = true }  # groups = ["user", "dev"]
```

and then run `poetry run pymedphys dev propagate`

Need to have the above syntax for Streamlit for two reasons, we need to pin Streamlit to an exact version given its API is in flux, and we need to have "optional = true" so that people can install PyMedPhys without installing Streamlit if they wish :).